### PR TITLE
Feature/myaccount

### PR DESCRIPTION
**Pull request by sullivan1lsu**
Button and Title before:
Snip20200122_4
Snip20200122_5

After:
Snip20200122_6
Snip20200122_7

### CODE REVIEW

File: our_files/custom.js [hunk 1]
```diff
@@ -2173,36 +2173,35 @@ Mobile Outgoing Markup:
 */
 
 var DetaildeUnavailableWhiteReserve = function() {
-  $J( document )
```
Comment (sullivan1lsu) [inline our_files/custom.js:2176]: Reformat, you can ignore

File: our_files/custom.js [hunk 2]
```diff
@@ -2267,36 +2266,35 @@ Mobile Outgoing Markup:
 */
 
 var deUnavailableReferenceMaterial = function() {
-  $J( document )
```
Comment (sullivan1lsu) [inline our_files/custom.js:2270]: Reformat, you can ignore

File: our_files/custom.js [hunk 3]
```diff
@@ -2360,36 +2358,35 @@ Mobile Outgoing Markup:
 */
 
 var deUnavailableReserveDesk = function() {
-  $J( document )
```
Comment (sullivan1lsu) [inline our_files/custom.js:2363]: Reformat, you can ignore

File: our_files/custom.js [hunk 4]
```diff
@@ -2592,18 +2589,75 @@ var hideBasicSearch = function() {
...
+/*
+Desktop Incoming Markup:  
+    <input 
+      onclick="$J(this).focus();openSmsPrefDialog('0', 'Add SMS Notification');return false;" 
+      id="addSmsPref" 
+      type="button" 
+      class="button fullwidth" 
+      value="Add SMS Notification">
 
-var changeSMSPopupTitle = function() {
```
Comment (sullivan1lsu) [inline our_files/custom.js:2603]: This item no longer exists.

File: our_files/custom.js [hunk 5]
```diff
@@ -2592,18 +2589,75 @@ var hideBasicSearch = function() {
...
+Desktop Outgoing Markup:
+      <input 
+      onclick="$J(this).focus();openSmsPrefDialog('0', 'Add Text Notification');return false;" 
+      id="addSmsPref" 
+      type="button" 
+      class="button fullwidth" 
+      value="Add Text Notification">
 
+*/
+var changeSMSTextButton = function() {
```
Comment (sullivan1lsu) [inline our_files/custom.js:2647]: New logic to handle the new button setup

**Comment by davidcomeaux** [comment]
Excellent! All looks good...

**Comment by sullivan1lsu** [comment]
@GarrettArm Merging, but please review this when you can

**Pull request merged**